src/soep_preparation/clean_existing_variables/hwealth.py

```diff
@@ -0,0 +1,174 @@
+"""Clean and convert SOEP hwealth variables to appropriate data types."""
+
+import pandas as pd
+
+from soep_preparation.utilities.series_manipulator import (
+    apply_smallest_float_dtype,
+    apply_smallest_int_dtype,
+    object_to_str_categorical,
+)
+
+
+def clean(raw_data: pd.DataFrame) -> pd.DataFrame:
+    """Create cleaned and sensible data type variables from the hwealth file.
+
+    Args:
+        raw_data: The raw hwealth data.
+
+    Returns:
+        The processed hwealth data.
+    """
+    wide = pd.DataFrame()
+    wide["tmp_survey_year"] = apply_smallest_int_dtype(raw_data["syear"])
```

Comment:
```suggestion
    wide["survey_year"] = apply_smallest_int_dtype(raw_data["syear"])
```
Not needed here, just when the name changes via wide_to_long.